app/settings.py

```diff
@@ -0,0 +1,20 @@
+import os
+from pathlib import Path
+
+BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+ENV_FILE = os.getenv('ENV_FILE', f'{BASE_DIR}/.env.local')
+if ENV_FILE:
+    from dotenv import load_dotenv
+
+    print(f"Running locally hence injecting env vars from {ENV_FILE}")
+    load_dotenv(ENV_FILE, override=True, verbose=True)
+
+AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID')
+AWS_SECRET_ACCESS_KEY = os.getenv('AWS_SECRET_ACCESS_KEY')
+AWS_S3_BUCKET_NAME = os.getenv('AWS_S3_BUCKET_NAME')
+AWS_S3_REGION_NAME = os.getenv('AWS_S3_REGION_NAME')
```

Comment:
I would suggest to use `os.environ[]` instead of `os.getenv()` here (`getenv()` return per default `None` when the key is not found). This way if they are not set we get automatically a better error message. Same for `AWS_DB_TABLE_NAME` and `AWS_DB_REGION_NAME`